src/main/java/github/grace000/erversay/RequestRouter.java

```diff
@@ -22,27 +29,33 @@ private Response getResponse(String path) {
             case PIGGLY_URI:
                 getResponseForPigglyGet();
                 break;
+            case METHODS_ONE_URI:
+                getResponseForOptions();
+                break;
             default:
                 getResponseForNotFound();
```

Comment:
If this is the default response from your server now, do you think that `DEFAULT_STATUS` is a good name for your 200 code?

Reply:
That's not a good name if only certain requests will get a 200 code. The default might actually be a 404. I'll rename that to be more descriptive.